app.rb

```diff
@@ -0,0 +1,129 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require_relative './classroom'
+
+class App
+  attr_accessor :books, :people, :rentals, :classroom
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @classroom = Classroom.new('Web Development')
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'Book has not been added yet, please create a book'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'Person has not been added yet, please create a person'
+    else
+      @people.each do |person|
+        puts "[#{person.class.name}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input a number]:'
+    person_type = gets.chomp.to_i
+
+    case person_type
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      invalid_input
+    end
+  end
+
+  def create_student
+    puts 'Age:'
+    age = gets.chomp.to_i
```

Comment:
[OPTIONAL]

- I added a word as the age, and the student still got created. I think it will be better if you handle this bug 👍🏽